server/registry/internal/storage/models/version.go

```diff
@@ -28,7 +28,7 @@ type Version struct {
 	Key         string    `gorm:"primaryKey"`
 	ProjectID   string    // Uniquely identifies a project.
 	ApiID       string    // Uniquely identifies an api within a project.
-	VersionID   string    // Uniquely identifies a version wihtin a api.
+	VersionID   string    // Uniquely identifies a version within a api.
```

Comment:
Technically this should be `an api`, but while it's theoretically possible for my tool to complain about this, I haven't tried encouraging it to do so, which means I'd only make the change if I were more awake than I am.

Reply:
Thanks, we'll address that separately.